kuma/users/admin.py

```diff
@@ -21,7 +21,8 @@ class UserAdmin(BaseUserAdmin):
 
 class UserBanAdmin(admin.ModelAdmin):
     fields = ('user', 'by', 'reason', 'is_active')
-    list_display = ('user', 'by', 'reason')
+    list_display = ('user', 'by', 'reason', 'is_active')
+    list_editable = ('is_active',)
```

Comment:
I don't think it is necessary for this PR but it was a cause of major confusion for me at first glance.